### PR TITLE
FISH-651 FISH-868 FISH-1504 FISH-1526 FISH-1527 Revert MicroProfile 4.0

### PR DESCRIPTION
**Pull request by Pandrex247**
This reverts https://github.com/payara/Payara-Enterprise-Documentation/pull/158

Main PR: https://github.com/payara/Payara-Enterprise/pull/425

### CODE REVIEW

File: docs/modules/ROOT/pages/documentation/microprofile/README.adoc [hunk 1]
```diff
@@ -129,6 +96,7 @@ supported by Payara Server and Payara Micro, an application targeting version
 |
 * JAX-RS 2.1
 * CDI 2.0
+* JSON-P 1.1
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/microprofile/README.adoc:99]:
```suggestion
```

File: docs/modules/ROOT/pages/documentation/microprofile/README.adoc [hunk 2]
```diff
@@ -156,6 +124,7 @@ supported by Payara Server and Payara Micro, an application targeting version
...
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/microprofile/README.adoc:127]:
```suggestion
```

File: docs/modules/ROOT/pages/documentation/microprofile/README.adoc [hunk 3]
```diff
@@ -183,6 +152,7 @@ supported by Payara Server and Payara Micro, an application targeting version
...
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/microprofile/README.adoc:155]:
```suggestion
```

**Review by OndroMih — CHANGES_REQUESTED** [review]
The duplicate JSON-P references don't have to be reverted too. Otherwise all looks good.

File: docs/modules/ROOT/pages/documentation/microprofile/faulttolerance.adoc [hunk 4]
```diff
@@ -20,10 +21,12 @@ detailed in the spec], the following is also implemented in Payara Server Enterp
...
+The executor service is used to execute methods annotated with `@Asynchronous`, whereas the scheduled executor
+service is used by the CircuitBreaker and Timeout interceptors for their timeout operations (scheduling the circuit breaker
+to be set to half open, and just timing out respectively).
 
 === Alternative @Asynchronous Annotations
-The Payara specific configuration property `MP_Fault_Tolerance_Alternative_Asynchronous_Annotations` can be used to specify a comma separated list of fully qualified class names of those annotations that should have the same effect as FT's `@Asynchronous`. These annotations do not have to be interceptor bindings.
+Since 5.192 the Payara specific configuration property `MP_Fault_Tolerance_Alternative_Asynchronous_Annotations` can be used to specify a comma separated list of fully qualified class names of those annotations that should have the same effect as FT's `@Asynchronous`. These annotations do not have to be interceptor bindings.
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/documentation/microprofile/faulttolerance.adoc:29]:
```suggestion
The the Payara specific configuration property `MP_Fault_Tolerance_Alternative_Asynchronous_Annotations` can be used to specify a comma separated list of fully qualified class names of those annotations that should have the same effect as FT's `@Asynchronous`. These annotations do not have to be interceptor bindings.
```
I think we don't need to revert this bit

File: docs/modules/ROOT/pages/documentation/microprofile/faulttolerance.adoc [hunk 5]
```diff
@@ -75,13 +79,15 @@ image:microprofile/fault-tolerance.png[Set Fault Tolerance Configuration]
 [[using-asadmin-commands]]
 === Using Asadmin Commands
 
+NOTE: Since Payara Platform Enterprise 5.20.0, the configuration no longer allows configuring user defined executor to run the fault tolerance execution. Instead, Fault Tolerance uses two dedicated executors that are well suited for the usage pattern typically experienced when executing Fault Tolerance semantics. It's possible to configure the maximum size of these pools to scale them up or down.
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/documentation/microprofile/faulttolerance.adoc:82]:
```suggestion
NOTE: The configuration no longer allows configuring user defined executor to run the fault tolerance execution. Instead, Fault Tolerance uses two dedicated executors that are well suited for the usage pattern typically experienced when executing Fault Tolerance semantics. It's possible to configure the maximum size of these pools to scale them up or down.
```

File: docs/modules/ROOT/pages/documentation/microprofile/jwt.adoc [hunk 6]
```diff
@@ -146,6 +146,8 @@ This properties file should contain the key `accepted.issuer` with as value the
 
 === Namespaced claims configuration
 
+_Since Payara Micro 5.184_
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/documentation/microprofile/jwt.adoc:149]:
```suggestion
```

**Review by AlanRoth — APPROVED** [review]
Just these nit picks

**Pull request merged**